### ISSUE TITLE
create a PyPi package

### ISSUE THREAD

**Issue by mikkokotila** [botlabio/autonomio#11]
Once the testing is done, we should have a PyPi package as we have everything ready for it. One thing that needs to be clear though is that for the Travis process, we don't use the PyPi package but do it from python setup.py install

**Comment by m-anish** [comment]
Agree! Do you want to do nightly builds for pypi or builds for every release. I guess I also need to research what would be preferable here.

**Comment by mikkokotila** [comment]
I think nightly builds will be good as I'm seeing a lot of people having issues where the pip package and master are not in sync. Or what do you think? I guess the other side is that if we take just from last stable release, then there is a little less stress. First seems to lead to more organized workflow, so ok, I vote for that.

**Comment by mikkokotila** [comment]
Ok great, with the latest version v.0.2.0 the time has come...to make a pypi package so that we can allow just pip install on the latest stable release.

**Comment by mikkokotila** [comment]
I think we're doing this last once everything else for the current milestone is done in the next 10 days or so. Then we make a release out of it, and have to remember to include setup.cfg to the repo root with: 

```
[metadata]
description-file = README.md
```

**Comment by mikkokotila** [comment]
This seems like the best way to do it: 

http://5minutes.youkidea.com/howto-deploy-python-package-on-pypi-with-github-and-travis.html
